packages/yew/src/virtual_dom/vlist.rs

```diff
@@ -202,6 +202,8 @@ mod feat_ssr {
                         .await;
                 }
                 _ => {
+                    // No idea why but clippy doesn't recognise the need for `w` to be mutable
+                    #[allow(clippy::needless_pass_by_ref_mut)]
```

Comment:
I cannot reproduce this lint. Can you try again with the latest stable (1.73.0)?

Reply:
Updated my rust-analyser and got rid of the bug, nice